proto/ptf/base_test.py

```diff
@@ -63,9 +60,7 @@ def stringify(n, length):
     'length' bytes, it is represented in the fewest number of bytes it
     does fit into without loss of precision.  It always returns a
     string at least one byte long, even if value=width=0."""
-    h = '%x' % n
-    s = ('0'*(len(h) % 2) + h).zfill(length*2).decode('hex')
-    return s
+    return int.to_bytes(n, length=length)
```

Comment:
only very recent versions of Python 3 (>= 3.11) have a default value for the `byteorder` argument.